src/retrieve-validation/get-boundary.ts

```diff
@@ -0,0 +1,12 @@
+// Helper function that produces a boundary for form data
+let boundary: null | string = null;
+
+const getBoundary = (): string => {
+	if (!boundary) {
+		boundary = `----CSSValidatorBoundary${String(Math.random()).slice(2)}`;
```

Comment:
This appears to be appending a random number rather than a random string with any alphanumeric characters. That significantly reduces the bits of entropy while still using the same number of characters.